fairlearn/reductions/_moments/moment.py

```diff
@@ -33,20 +39,27 @@ def load_data(self, X, y, **kwargs):
         assert self.data_loaded is False, \
             "data can be loaded only once"
         self.X = X
-        self.n = self.X.shape[0]
         self.tags = pd.DataFrame({_LABEL: y})
         if _KW_SENSITIVE_FEATURES in kwargs:
             self.tags[_GROUP_ID] = kwargs[_KW_SENSITIVE_FEATURES]
         self.data_loaded = True
         self._gamma_descr = None
 
+    @property
+    def n_samples(self):
+        """Return the number of samples in the data."""
```

Comment:
since we are renaming this, how about something like "total_samples" instead of "n_samples"? this would more clearly work in the various expressions when we calculate probabilities since it would be more obvious that we are dividing "number of samples with a specific property" by the "total number of samples".

Reply:
Done. I renamed it since trying to track down something just called `n` is not fun :-)